pennylane/fermi/conversion.py

```diff
@@ -164,6 +164,11 @@ def parity_transform(
 ) -> Union[Operator, PauliSentence]:
     r"""Convert a fermionic operator to a qubit operator using the parity mapping.
 
+    .. note::
+
+        Hamiltonians created with this mapping should be only used with operators and states in the
+        parity basis.
+
```

Comment:
So does this mean that every qubit operator you create has to have come via `parity_transform`?

Reply:
If the operator is a fermionic operator mapped to a qubit operator, then the mapping method should match the one used for the Hamiltonian.